data.py-dist

```diff
@@ -50,14 +50,32 @@ VM_IMAGES = {
 }
 
 # FIXME: use URLs and an optional cache?
+#
+# If 'net-only' is set to 'True' only source of type URL will be possible.
+# By default the parameter is set to False.
 ISO_IMAGES = {
-#    'xcpng-8.0': "/home/user/iso/xcp-ng-8.0.0.iso",
-#    'xcpng-8.1': "/home/user/iso/xcp-ng-8.1.0-2.iso",
-#    'xcpng-8.2.0': "/home/user/iso/xcp-ng-8.2.0.iso",
-#    'xcpng-8.2.1': "/home/user/iso/xcp-ng-8.2.1.iso",
-#    'xcpng-8.2.1-2023': "/home/user/iso/xcp-ng-8.2.1-20231130.iso",
-#    'xcpng-8.3-beta2': "/home/user/iso/xcp-ng-8.3.0-beta2.iso",
-#    'xcpng-8.3-beta2-net': "/home/user/iso/xcp-ng-8.3.0-beta2-netinstall.iso",
+#    'xcpng-8.0': {'path': "/home/user/iso/xcp-ng-8.0.0.iso"},
+#    'xcpng-8.1': {'path': "/home/user/iso/xcp-ng-8.1.0-2.iso"},
+#    'xcpng-8.2.0': {'path': "/home/user/iso/xcp-ng-8.2.0.iso"},
+#    'xcpng-8.2.1': {'path': "/home/user/iso/xcp-ng-8.2.1.iso"},
+#    'xcpng-8.2.1-2023': {'path': "/home/user/iso/xcp-ng-8.2.1-20231130.iso"},
+#    'xcpng-8.3-beta2': {
+#       'path': "/home/user/iso/xcp-ng-8.3.0-beta2.iso"
+#       'net-url': "http://server/installers/xcp-ng/8.3-beta2",
+#    },
+#    'xcpng-8.3-beta2-net': {
+#       'path': "/home/user/iso/xcp-ng-8.3.0-beta2-netinstall.iso",
+#       'net-url': "http://server/installers/xcp-ng/8.3-beta2",
+#       'net-only': True,
+#       'pxe': {
+#         'url': "http://server/installers/xcp-ng/8.3-beta2",
+#         'script': "http://server/configs/presets/scripts/filesystem-populated.py",
+#         'upload': "http://server/configs/custom",
```

Comment:
currently it is hardcoded in `lib/pxe.py` so maybe we can remove it. And it is the same for the script `filesystem-populated so we probably need to only keep the url.